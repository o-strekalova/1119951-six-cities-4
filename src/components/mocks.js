export const offersAll = [{
  id: `1010`,
  city: {
    location: {
      lat: 52.3909553943508,
      long: 4.85309666406198,
      zoom: 13,
    },
    name: `Amsterdam`,
  },
  location: {
    lat: 52.3909553943508,
    long: 4.85309666406198,
    zoom: 13,
  },
  pictures: [`img/apartment-01.jpg`, `img/apartment-02.jpg`, `img/apartment-03.jpg`, `img/room.jpg`, `img/apartment-01.jpg`, `img/apartment-02.jpg`],
  preview: `img/apartment-01.jpg`,
  title: `Wood and stone place`,
  type: `room`,
  price: 999,
  isPremium: true,
  isFavorite: false,
  rating: 4.4,
  description: ``,
  bedrooms: 2,
  guests: 4,
  features: [],
  owner: {
    avatar: `https://api.adorable.io/avatars/128`,
    name: ``,
    isSuper: true,
    id: 111,
  },
}, {
  id: `2020`,
  city: {
    location: {
      lat: 52.3909553943508,
      long: 4.85309666406198,
      zoom: 13,
    },
    name: `Paris`,
  },
  location: {
    lat: 52.3909553943508,
    long: 4.85309666406198,
    zoom: 13,
  },
  pictures: [`img/apartment-01.jpg`, `img/apartment-02.jpg`, `img/apartment-03.jpg`, `img/room.jpg`, `img/apartment-01.jpg`, `img/apartment-02.jpg`],
  preview: `img/apartment-01.jpg`,
  title: `Wood and stone place`,
  type: `room`,
  price: 999,
  isFavorite: false,
  isPremium: true,
  rating: 4.4,
  description: ``,
  bedrooms: 2,
  guests: 4,
  features: [],
  owner: {
    avatar: `https://api.adorable.io/avatars/128`,
    name: ``,
    isSuper: true,
    id: 222,
  },
}];

export const offersRaw = [{
  "id": 1010,
  "city": {
    "location": {
      "latitude": 52.3909553943508,
      "longitude": 4.85309666406198,
      "zoom": 13,
    },
    "name": `Amsterdam`,
  },
  "location": {
    "latitude": 52.3909553943508,
    "longitude": 4.85309666406198,
    "zoom": 13,
  },
  "images": [`img/apartment-01.jpg`, `img/apartment-02.jpg`, `img/apartment-03.jpg`, `img/room.jpg`, `img/apartment-01.jpg`, `img/apartment-02.jpg`],
  "preview_image": `img/apartment-01.jpg`,
  "title": `Wood and stone place`,
  "type": `room`,
  "price": 999,
  "is_premium": true,
  "is_favorite": false,
  "rating": 4.4,
  "description": ``,
  "bedrooms": 2,
  "max_adults": 4,
  "goods": [],
  "host": {
    "avatar_url": `https://api.adorable.io/avatars/128`,
    "name": ``,
    "is_pro": true,
    "id": 111,
  },
}, {
  "id": 2020,
  "city": {
    "location": {
      "latitude": 52.3909553943508,
      "longitude": 4.85309666406198,
      "zoom": 13,
    },
    "name": `Paris`,
  },
  "location": {
    "latitude": 52.3909553943508,
    "longitude": 4.85309666406198,
    "zoom": 13,
  },
  "images": [`img/apartment-01.jpg`, `img/apartment-02.jpg`, `img/apartment-03.jpg`, `img/room.jpg`, `img/apartment-01.jpg`, `img/apartment-02.jpg`],
  "preview_image": `img/apartment-01.jpg`,
  "title": `Wood and stone place`,
  "type": `room`,
  "price": 999,
  "is_favorite": false,
  "is_premium": true,
  "rating": 4.4,
  "description": ``,
  "bedrooms": 2,
  "max_adults": 4,
  "goods": [],
  "host": {
    "avatar_url": `https://api.adorable.io/avatars/128`,
    "name": ``,
    "is_pro": true,
    "id": 222,
  },
}];

export const reviews = [{
  id: 1110,
  avatar: `https://api.adorable.io/avatars/128`,
  name: `Angelina`,
  rating: 4.4,
  date: new Date(2020, 10, 5, 10),
  text: `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.`,
}];

export const authInfo = {
  avatar: `https://api.adorable.io/avatars/128`,
  email: `mail@mail.ru`,
  id: 1,
  isSuper: true,
  name: `Angelina`,
};
