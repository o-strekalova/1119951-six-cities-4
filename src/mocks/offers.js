const OFFERS_TITLES = [`Beautiful & luxurious apartment at great location`, `Wood and stone place`, `Canal View Prinsengracht`, `Nice, cozy, warm big bed apartment`];
const OFFERS_TYPES = [`apartment`, `room`, `house`, `hotel`];
const AVATAR_URL = `https://api.adorable.io/avatars/128`;
const COORDINATES = [
  [52.3909553943508, 4.85309666406198],
  [52.369553943508, 4.85309666406198],
  [52.3909553943508, 4.929309666406198],
  [52.3809553943508, 4.939309666406198]
];
const OFFERS_COUNT = 4;

const getRandomIntegerNumber = (min, max) => {
  return min + Math.floor(Math.random() * (max - min));
};

const getRandomArrayItem = (array) => {
  const randomIndex = getRandomIntegerNumber(0, array.length);
  return array[randomIndex];
};

const generateOffers = () => {
  const offers = [];

  for (let i = 0; i < OFFERS_COUNT; i++) {
    offers.push({
      id: i,
      coords: COORDINATES[i],
      pictures: [`img/apartment-01.jpg`, `img/apartment-02.jpg`, `img/apartment-03.jpg`, `img/room.jpg`, `img/apartment-01.jpg`, `img/apartment-02.jpg`],
      title: getRandomArrayItem(OFFERS_TITLES),
      type: getRandomArrayItem(OFFERS_TYPES),
      price: getRandomIntegerNumber(50, 1000),
      isPremium: Math.random() > 0.5,
      rating: getRandomIntegerNumber(0, 4) + Math.round(Math.random() * 10) / 10,
      description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.`,
      bedrooms: getRandomIntegerNumber(1, 10),
      guests: getRandomIntegerNumber(1, 20),
      features: [`Wi-Fi`, `Washing machine`, `Towels`, `Heating`, `Coffee machine`, `Baby seat`, `Kitchen`, `Dishwasher`, `Cabel TV`, `Fridge`],
      owner: {
        avatar: AVATAR_URL,
        name: `Angelina`,
        isSuper: Math.random() > 0.5,
      },
    });
  }

  return offers;
};

export const offers = generateOffers();
