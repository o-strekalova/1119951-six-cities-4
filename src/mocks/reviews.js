const AVATAR_URL = `https://api.adorable.io/avatars/128`;
const REVIEWS_COUNT = 11;

const getRandomIntegerNumber = (min, max) => {
  return min + Math.floor(Math.random() * (max - min));
};

const generateReviews = () => {
  const reviews = [];

  for (let i = 0; i < REVIEWS_COUNT; i++) {
    reviews.push({
      id: i,
      avatar: AVATAR_URL,
      name: `Angelina`,
      rating: getRandomIntegerNumber(0, 4) + Math.round(Math.random() * 10) / 10,
      date: new Date(),
      text: `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.`,
    });
  }

  return reviews;
};

export const reviews = generateReviews();
