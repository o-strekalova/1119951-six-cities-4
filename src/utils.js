export const CardsListClass = {
  MAIN: `cities__places-list tabs__content`,
  PROPERTY: `near-places__list`,
};

export const CardClass = {
  MAIN: `cities__place-card`,
  PROPERTY: `near-places__card`,
};

export const SortType = {
  POPULAR: `Popular`,
  PRICE_FROM_LOW: `Price: low to high`,
  PRICE_FROM_HIGH: `Price: high to low`,
  TOP_RATED: `Top rated first`,
};

export const extend = (a, b) => {
  return Object.assign({}, a, b);
};

export const getRatingPercentage = (rating) => {
  return Math.round(rating) * 20;
};
