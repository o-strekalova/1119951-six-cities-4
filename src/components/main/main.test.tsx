import * as React from "react";
import * as renderer from "react-test-renderer";
import {Router} from "react-router-dom";
import Main from "./main";
import {offersAll, authInfo} from "../mocks";
import {SortType, AuthorizationStatus, noop} from "../../utils";
import history from "../../history";

it(`Render Main with offers`, () => {
  const tree = renderer
    .create(
        <Router
          history={history}
        >
          <Main
            authInfo={authInfo}
            authorizationStatus={AuthorizationStatus.AUTH}
            errorMessage={null}
            offersAll={offersAll}
            activeCity={offersAll[0].city}
            sortedOffers={offersAll}
            activeSort={SortType.POPULAR}
            onCardTitleClick={noop}
            onCityClick={noop}
            onSortClick={noop}
            onAuthFormSubmit={noop}
            onFavoriteButtonClick={noop}
            onUserNameClick={noop}
          />
        </Router>, {
          createNodeMock: () => {
            return document.createElement(`DIV`);
          }
        })
    .toJSON();

  expect(tree).toMatchSnapshot();
});

it(`Render Main without offers`, () => {
  const tree = renderer
    .create(
        <Router
          history={history}
        >
          <Main
            authInfo={authInfo}
            authorizationStatus={AuthorizationStatus.AUTH}
            errorMessage={null}
            offersAll={[]}
            activeCity={offersAll[0].city}
            sortedOffers={[]}
            activeSort={SortType.POPULAR}
            onCardTitleClick={noop}
            onCityClick={noop}
            onSortClick={noop}
            onAuthFormSubmit={noop}
            onFavoriteButtonClick={noop}
            onUserNameClick={noop}
          />
        </Router>, {
          createNodeMock: () => {
            return document.createElement(`DIV`);
          }
        })
    .toJSON();

  expect(tree).toMatchSnapshot();
});

it(`Render Main for unauthorized user`, () => {
  const tree = renderer
    .create(
        <Router
          history={history}
        >
          <Main
            authInfo={null}
            authorizationStatus={AuthorizationStatus.NO_AUTH}
            errorMessage={null}
            offersAll={[]}
            activeCity={offersAll[0].city}
            sortedOffers={[]}
            activeSort={SortType.POPULAR}
            onCardTitleClick={noop}
            onCityClick={noop}
            onSortClick={noop}
            onAuthFormSubmit={noop}
            onFavoriteButtonClick={noop}
            onUserNameClick={noop}
          />
        </Router>, {
          createNodeMock: () => {
            return document.createElement(`DIV`);
          }
        })
    .toJSON();

  expect(tree).toMatchSnapshot();
});

it(`Render Main for with ErrorMessage`, () => {
  const tree = renderer
    .create(
        <Router
          history={history}
        >
          <Main
            authInfo={null}
            authorizationStatus={AuthorizationStatus.NO_AUTH}
            errorMessage={`Failed to load offers`}
            offersAll={[]}
            activeCity={offersAll[0].city}
            sortedOffers={[]}
            activeSort={SortType.POPULAR}
            onCardTitleClick={noop}
            onCityClick={noop}
            onSortClick={noop}
            onAuthFormSubmit={noop}
            onFavoriteButtonClick={noop}
            onUserNameClick={noop}
          />
        </Router>, {
          createNodeMock: () => {
            return document.createElement(`DIV`);
          }
        })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
