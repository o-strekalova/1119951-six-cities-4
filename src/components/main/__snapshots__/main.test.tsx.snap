// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Main for unauthorized user 1`] = `
<div
  className="page page--gray page--main"
>
  <header
    className="header"
    id="header"
  >
    <div
      className="container"
    >
      <div
        className="header__wrapper"
      >
        <div
          className="header__left"
        >
          <a
            className="header__logo-link"
            href="/"
            onClick={[Function]}
          >
            <img
              alt="6 cities logo"
              className="header__logo"
              height="41"
              src="img/logo.svg"
              width="81"
            />
          </a>
        </div>
        <nav
          className="header__nav"
        >
          <ul
            className="header__nav-list"
          >
            <li
              className="header__nav-item user"
            >
              <span
                className="header__nav-link header__nav-link--profile"
              >
                <div
                  className="header__avatar-wrapper user__avatar-wrapper"
                />
                <a
                  className="header__login"
                  href="/login"
                  onClick={[Function]}
                >
                  Sign in
                </a>
              </span>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </header>
  <main
    className="page__main page__main--index page__main--index-empty"
  >
    <h1
      className="visually-hidden"
    >
      Cities
    </h1>
    <div
      className="tabs"
    >
      <section
        className="locations container"
      >
        <ul
          className="locations__list tabs__list"
        />
      </section>
    </div>
    <div
      className="cities"
    >
      <div
        className="cities__places-container cities__places-container--empty container"
      >
        <section
          className="cities__no-places"
        >
          <div
            className="cities__status-wrapper tabs__content"
          >
            <b
              className="cities__status"
            >
              No places to stay available
            </b>
            <p
              className="cities__status-description"
            >
              We could not find any property availbale at the moment
               in Amsterdam
            </p>
          </div>
        </section>
        <div
          className="cities__right-section"
        />
      </div>
    </div>
  </main>
</div>
`;

exports[`Render Main for with ErrorMessage 1`] = `
<div
  className="page page--gray page--main"
>
  <header
    className="header"
    id="header"
  >
    <div
      className="container"
    >
      <div
        className="header__wrapper"
      >
        <div
          className="header__left"
        >
          <a
            className="header__logo-link"
            href="/"
            onClick={[Function]}
          >
            <img
              alt="6 cities logo"
              className="header__logo"
              height="41"
              src="img/logo.svg"
              width="81"
            />
          </a>
        </div>
        <nav
          className="header__nav"
        >
          <ul
            className="header__nav-list"
          >
            <li
              className="header__nav-item user"
            >
              <span
                className="header__nav-link header__nav-link--profile"
              >
                <div
                  className="header__avatar-wrapper user__avatar-wrapper"
                />
                <a
                  className="header__login"
                  href="/login"
                  onClick={[Function]}
                >
                  Sign in
                </a>
              </span>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </header>
  <main
    className="page__main page__main--index page__main--index-empty"
  >
    <h1
      className="visually-hidden"
    >
      Cities
    </h1>
    <div
      className="tabs"
    >
      <section
        className="locations container"
      >
        <ul
          className="locations__list tabs__list"
        />
      </section>
    </div>
    <div
      className="cities"
    >
      <div
        className="cities__places-container cities__places-container--empty container"
      >
        <section
          className="cities__no-places"
        >
          <div
            className="cities__status-wrapper tabs__content"
          >
            <b
              className="cities__status"
            >
              No places to stay available
            </b>
            <p
              className="cities__status-description"
            >
              We could not find any property availbale at the moment
               in Amsterdam
            </p>
          </div>
        </section>
        <div
          className="cities__right-section"
        />
      </div>
    </div>
  </main>
  <div
    style={
      Object {
        "backgroundColor": "red",
        "color": "white",
        "left": 0,
        "margin": "auto",
        "padding": "10px 20px",
        "position": "fixed",
        "right": 0,
        "textAlign": "center",
        "top": 0,
        "width": "200px",
      }
    }
  >
    Failed to load offers
  </div>
</div>
`;

exports[`Render Main with offers 1`] = `
<div
  className="page page--gray page--main"
>
  <header
    className="header"
    id="header"
  >
    <div
      className="container"
    >
      <div
        className="header__wrapper"
      >
        <div
          className="header__left"
        >
          <a
            className="header__logo-linkheader__logo-link--active"
            href="/"
            onClick={[Function]}
          >
            <img
              alt="6 cities logo"
              className="header__logo"
              height="41"
              src="img/logo.svg"
              width="81"
            />
          </a>
        </div>
        <nav
          className="header__nav"
        >
          <ul
            className="header__nav-list"
          >
            <li
              className="header__nav-item user"
            >
              <span
                className="header__nav-link header__nav-link--profile"
              >
                <div
                  className="header__avatar-wrapper user__avatar-wrapper"
                />
                <a
                  className="header__user-name user__name"
                  href="/favorites"
                  onClick={[Function]}
                >
                  mail@mail.ru
                </a>
              </span>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </header>
  <main
    className="page__main page__main--index"
  >
    <h1
      className="visually-hidden"
    >
      Cities
    </h1>
    <div
      className="tabs"
    >
      <section
        className="locations container"
      >
        <ul
          className="locations__list tabs__list"
        >
          <li
            className="locations__item"
            onClick={[Function]}
          >
            <a
              className="locations__item-link tabs__item tabs__item--active"
              href="#"
            >
              <span>
                Amsterdam
              </span>
            </a>
          </li>
          <li
            className="locations__item"
            onClick={[Function]}
          >
            <a
              className="locations__item-link tabs__item"
              href="#"
            >
              <span>
                Paris
              </span>
            </a>
          </li>
        </ul>
      </section>
    </div>
    <div
      className="cities"
    >
      <div
        className="cities__places-container container"
      >
        <section
          className="cities__places places"
        >
          <h2
            className="visually-hidden"
          >
            Places
          </h2>
          <b
            className="places__found"
          >
            2
             places to stay in 
            Amsterdam
          </b>
          <form
            action="#"
            className="places__sorting"
            method="get"
          >
            <span
              className="places__sorting-caption"
            >
              Sort by
            </span>
            <span
              className="places__sorting-type"
              onClick={[Function]}
              tabIndex={0}
            >
              Popular
              <svg
                className="places__sorting-arrow"
                height="4"
                width="7"
              >
                <use
                  xlinkHref="#icon-arrow-select"
                />
              </svg>
            </span>
          </form>
          <div
            className="cities__places-list tabs__content places__list"
          >
            <article
              className="cities__place-card place-card"
              onMouseOver={[Function]}
            >
              <div
                className="place-card__mark"
              >
                <span>
                  Premium
                </span>
              </div>
              <div
                className="cities__image-wrapper place-card__image-wrapper"
              >
                <a
                  href="#"
                >
                  <img
                    alt="Wood and stone place"
                    className="place-card__image"
                    height="200"
                    src="img/apartment-01.jpg"
                    width="260"
                  />
                </a>
              </div>
              <div
                className="place-card__info"
              >
                <div
                  className="place-card__price-wrapper"
                >
                  <div
                    className="place-card__price"
                  >
                    <b
                      className="place-card__price-value"
                    >
                      €
                      999
                    </b>
                    <span
                      className="place-card__price-text"
                    >
                      / night
                    </span>
                  </div>
                  <button
                    className="place-card__bookmark-button button"
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      className="place-card__bookmark-icon"
                      height="19"
                      width="18"
                    >
                      <use
                        xlinkHref="#icon-bookmark"
                      />
                    </svg>
                    <span
                      className="visually-hidden"
                    >
                      To bookmarks
                    </span>
                  </button>
                </div>
                <div
                  className="place-card__rating rating"
                >
                  <div
                    className="place-card__stars rating__stars"
                  >
                    <span
                      style={
                        Object {
                          "width": "80%",
                        }
                      }
                    />
                    <span
                      className="visually-hidden"
                    >
                      Rating
                    </span>
                  </div>
                </div>
                <h2
                  className="place-card__name"
                >
                  <a
                    href="/offer/1010/#header"
                    onClick={[Function]}
                  >
                    Wood and stone place
                  </a>
                </h2>
                <p
                  className="place-card__type"
                >
                  Room
                </p>
              </div>
            </article>
            <article
              className="cities__place-card place-card"
              onMouseOver={[Function]}
            >
              <div
                className="place-card__mark"
              >
                <span>
                  Premium
                </span>
              </div>
              <div
                className="cities__image-wrapper place-card__image-wrapper"
              >
                <a
                  href="#"
                >
                  <img
                    alt="Wood and stone place"
                    className="place-card__image"
                    height="200"
                    src="img/apartment-01.jpg"
                    width="260"
                  />
                </a>
              </div>
              <div
                className="place-card__info"
              >
                <div
                  className="place-card__price-wrapper"
                >
                  <div
                    className="place-card__price"
                  >
                    <b
                      className="place-card__price-value"
                    >
                      €
                      999
                    </b>
                    <span
                      className="place-card__price-text"
                    >
                      / night
                    </span>
                  </div>
                  <button
                    className="place-card__bookmark-button button"
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      className="place-card__bookmark-icon"
                      height="19"
                      width="18"
                    >
                      <use
                        xlinkHref="#icon-bookmark"
                      />
                    </svg>
                    <span
                      className="visually-hidden"
                    >
                      To bookmarks
                    </span>
                  </button>
                </div>
                <div
                  className="place-card__rating rating"
                >
                  <div
                    className="place-card__stars rating__stars"
                  >
                    <span
                      style={
                        Object {
                          "width": "80%",
                        }
                      }
                    />
                    <span
                      className="visually-hidden"
                    >
                      Rating
                    </span>
                  </div>
                </div>
                <h2
                  className="place-card__name"
                >
                  <a
                    href="/offer/2020/#header"
                    onClick={[Function]}
                  >
                    Wood and stone place
                  </a>
                </h2>
                <p
                  className="place-card__type"
                >
                  Room
                </p>
              </div>
            </article>
          </div>
        </section>
        <div
          className="cities__right-section"
        >
          <section
            className="cities__map map"
          >
            <div
              id="map"
              style={
                Object {
                  "height": "100%",
                }
              }
            />
          </section>
        </div>
      </div>
    </div>
  </main>
</div>
`;

exports[`Render Main without offers 1`] = `
<div
  className="page page--gray page--main"
>
  <header
    className="header"
    id="header"
  >
    <div
      className="container"
    >
      <div
        className="header__wrapper"
      >
        <div
          className="header__left"
        >
          <a
            className="header__logo-linkheader__logo-link--active"
            href="/"
            onClick={[Function]}
          >
            <img
              alt="6 cities logo"
              className="header__logo"
              height="41"
              src="img/logo.svg"
              width="81"
            />
          </a>
        </div>
        <nav
          className="header__nav"
        >
          <ul
            className="header__nav-list"
          >
            <li
              className="header__nav-item user"
            >
              <span
                className="header__nav-link header__nav-link--profile"
              >
                <div
                  className="header__avatar-wrapper user__avatar-wrapper"
                />
                <a
                  className="header__user-name user__name"
                  href="/favorites"
                  onClick={[Function]}
                >
                  mail@mail.ru
                </a>
              </span>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </header>
  <main
    className="page__main page__main--index page__main--index-empty"
  >
    <h1
      className="visually-hidden"
    >
      Cities
    </h1>
    <div
      className="tabs"
    >
      <section
        className="locations container"
      >
        <ul
          className="locations__list tabs__list"
        />
      </section>
    </div>
    <div
      className="cities"
    >
      <div
        className="cities__places-container cities__places-container--empty container"
      >
        <section
          className="cities__no-places"
        >
          <div
            className="cities__status-wrapper tabs__content"
          >
            <b
              className="cities__status"
            >
              No places to stay available
            </b>
            <p
              className="cities__status-description"
            >
              We could not find any property availbale at the moment
               in Amsterdam
            </p>
          </div>
        </section>
        <div
          className="cities__right-section"
        />
      </div>
    </div>
  </main>
</div>
`;
